Microsoft.Azure.CosmosRepository/src/IRepository.cs

```diff
@@ -31,6 +32,21 @@ namespace Microsoft.Azure.CosmosRepository
     /// </example>
     public interface IRepository<TItem> where TItem : IItem
     {
+
+        /// <summary>
+        /// Pages results of a given item.
+        /// </summary>
+        /// <param name="predicate"></param>
+        /// <param name="cancellationToken"></param>
+        /// <param name="pageSize"></param>
+        /// <param name="page"></param>
+        /// <param name="continuationToken"></param>
+        /// <returns></returns>
+        ValueTask<IPage<TItem>> PageAsync(Expression<Func<TItem, bool>> predicate,
+            CancellationToken cancellationToken = default, int pageSize = 25, int page = 1,
+            string continuationToken = null);

```

Comment:
Haven't read through the rest of the API, but should this be `string? continuationToken = null`?